verify_search_clear.py

```
@@ -0,0 +1,62 @@
+def verify_clear_button():
+    with sync_playwright() as p:
+        browser = p.chromium.launch(headless=True)
+        context = browser.new_context()
+        page = context.new_page()
+
+        print("Navigating to Flow Studio...")
+        try:
+            page.goto("http://localhost:5000")
+            page.wait_for_selector('html[data-ui-ready="ready"]', timeout=10000)
+            print("UI ready.")
+        except Exception as e:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)